### ISSUE TITLE
"SGIS_texture_edge_clamp unavailable" but "GL_EXT_texture_edge_clamp: OK"

### ISSUE THREAD

**Issue by BlaXpirit** [SFML/SFML#880]
OpenGL extension SGIS_texture_edge_clamp unavailable
    Artifacts may occur along texture edges

Many people seem to be getting this warning now. And I have indeed noticed artifacts along texture edges, but the same happened in older versions. As I understand it, the problem was always there, just that the warning message appeared only in SFML 2.3. 

I have this on an old NVIDIA graphics card, other people reported the same on new ones.

And the thing is...

```bash
glewinfo | grep texture_edge_clamp
```

    GL_EXT_texture_edge_clamp:                                     OK 
    GL_SGIS_texture_edge_clamp:                                    MISSING 

Some graphics cards seem to have this feature under a different name. It may be worth adding a fallback like this.

**Comment by binary1248** [comment]
Hmm... I guess Nvidia doesn't think it makes sense to expose the same extension under its aliases like everybody else does. Will fix.

> As I understand it, the problem was always there, just that the warning message appeared only in SFML 2.3.

It definitely wasn't always there, because there was never a problem. :wink:

This "issue" has only became apparent now after we added support for hardware that doesn't support texture edge clamping, because I guess not many people with Nvidia cards have tested/reported the issue after the commit got merged.
